### ISSUE TITLE
Error Parsing GraphQL query (Non-ASCII characters)

### ISSUE THREAD

**Issue by filipemacedo** [prisma/prisma#21578]
### Bug description

Hello.

Recently I received this error on my application when a user tried to create an entity. I reproduced the behavior on Prisma Playground, basically if the payload contains this string:

```
⚪️\ud83d
```

If you can create an entity with this string, you'll probably receive this error:

```
Error parsing GraphQL query: query parse error: Parse error at 4:11
Unexpected d83d is not a valid Unicode code point
```

Image

![image](https://github.com/prisma/prisma/assets/28747696/4afbb2a8-e6f7-4415-a81f-a46f7a7622f8)

### Expected behavior

The expected behavior is a successful response.

### Prisma information

You can reproduce the problem using the Prisma Playground.

### Environment & setup

Prisma Playground Setup

### Prisma Version

Prisma Playground Version

**Comment by janpio** [comment]
Were you getting the same error in your local Prisma Client?
What version of Prisma Client were you using there?

**Comment by filipemacedo** [comment]
> Were you getting the same error in your local Prisma Client? What version of Prisma Client were you using there?

Yes. We're using this version: 4.1.1

I'll try to upgrade the version to test again, but when I tested on Playground and received the same error I thought it was better to report it.

**Comment by janpio** [comment]
Absolutely, thanks! Playground is running 4.7.0 (I found out), so maybe this is already fixed in 5.x. So optimally we can already close this without any action needed.

**Comment by Jolg42** [comment]
@filipemacedo could you provide the actual "surrogate pair"?
Here in `name: "Hello \ud83d",` it's missing one part.

**Comment by filipemacedo** [comment]
Hi @Jolg42. Sorry, but this was the user's request data
